lomiri.lomiri-schemas: 0.1.5 -> 0.1.6

OPNA2608 opened the pull request:
https://gitlab.com/ubports/development/core/lomiri-schemas/-/blob/0.1.6/ChangeLog

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

OPNA2608 committed:
lomiri.lomiri-schemas: 0.1.5 -> 0.1.6

OPNA2608 commented:
This makes Lomiri crash on startup, in [LAL's `findExistingIcon`](https://gitlab.com/ubports/development/core/lomiri-app-launch/-/blob/a057a37b56b96efddc7f42577407a3a08a0575cf/liblomiri-app-launch/application-icon-finder.cpp#L110). Since it should find [LSS's desktop file](https://gitlab.com/ubports/development/core/lomiri-schemas/-/commit/a1ebd43938652c21e6230bfaf1eabcb3c36e1e70#5aeec74b0c0766cd7fc4d4eecd9b453e6172a5e1_278_277) now, maybe this is the same issue that Morph Browser was having/causing:

https://github.com/NixOS/nixpkgs/blob/6df24922a1400241dae323af55f30e4318a6ca65/pkgs/desktops/lomiri/applications/morph-browser/default.nix#L54-L59

I guess now's the time to look into what's going wrong in LAL, and fix it…

OPNA2608 committed:
lomiri.lomiri-app-launch: Fix crash on Lomiri startup when resolving relative icons for pinned apps

OPNA2608 committed:
lomiri.morph-browser: Drop workaround for LAL crash on Lomiri startup

lucasew commented on pkgs/desktops/lomiri/development/lomiri-app-launch/default.nix:
```diff
@@ -58,6 +58,13 @@ stdenv.mkDerivation (finalAttrs: {
       hash = "sha256-11pEhFi39Cvqb9Hg47kT8+5hq+bz6WmySqaIdwt1MVk=";
     })
 
+    # Remove when https://gitlab.com/ubports/development/core/lomiri-app-launch/-/merge_requests/65 merged & in release
```
Please download and vendor the patch.

There is rising concerns about the availability of the URL, if the patch is in nixpkgs directly this is not a problem.

OPNA2608 replied:
Any issue or discussion threads on this? Pretty sure we used to *not* vendor patches when they could instead be sent to upstream & fetched from there.

OPNA2608 replied:
There's a section in our contributing document specifically for this: https://github.com/NixOS/nixpkgs/blob/2c8f8a719a9018a43984062272ebf5772dc786d6/pkgs/README.md#vendoring-patches

> In the following cases, a `.patch` file _should_ be added to Nixpkgs repository, instead of retrieved:
> 
> […]
> 
> - is already proposed upstream but not merged yet

Just linking that would've been enough to clear that up for me.

---

Since then, both MRs have been accepted & merged, so this is all not relevant for this PR anymore.

lucasew commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 370966`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>ayatana-indicator-sound</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 38 packages built:</summary>
  <ul>
    <li>ayatana-indicator-bluetooth</li>
    <li>ayatana-indicator-datetime</li>
    <li>ayatana-indicator-display</li>
    <li>ayatana-indicator-power</li>
    <li>ayatana-indicator-session</li>
    <li>deltatouch</li>
    <li>libayatana-common</li>
    <li>lomiri.lomiri</li>
    <li>lomiri.lomiri-app-launch</li>
    <li>lomiri.lomiri-app-launch.dev</li>
    <li>lomiri.lomiri-app-launch.doc</li>
    <li>lomiri.lomiri-camera-app</li>
    <li>lomiri.lomiri-clock-app</li>
    <li>lomiri.lomiri-content-hub</li>
    <li>lomiri.lomiri-content-hub.dev</li>
    <li>lomiri.lomiri-content-hub.doc</li>
    <li>lomiri.lomiri-content-hub.examples</li>
    <li>lomiri.lomiri-docviewer-app</li>
    <li>lomiri.lomiri-filemanager-app</li>
    <li>lomiri.lomiri-gallery-app</li>
    <li>lomiri.lomiri-indicator-network</li>
    <li>lomiri.lomiri-indicator-network.dev</li>
    <li>lomiri.lomiri-indicator-network.doc</li>
    <li>lomiri.lomiri-mediaplayer-app</li>
    <li>lomiri.lomiri-push-qml</li>
    <li>lomiri.lomiri-schemas</li>
    <li>lomiri.lomiri-session</li>
    <li>lomiri.lomiri-system-settings</li>
    <li>lomiri.lomiri-system-settings-unwrapped</li>
    <li>lomiri.lomiri-system-settings-unwrapped.dev</li>
    <li>lomiri.lomiri-url-dispatcher</li>
    <li>lomiri.lomiri-url-dispatcher.dev</li>
    <li>lomiri.lomiri-url-dispatcher.lib</li>
    <li>lomiri.morph-browser</li>
    <li>lomiri.qtmir</li>
    <li>lomiri.qtmir.dev</li>
    <li>lomiri.telephony-service</li>
    <li>lomiri.teleports</li>
  </ul>
</details>

OPNA2608 commented:
`ayatana-indicator-sound` failure I assume is the same as in #369525? Any details on that yet?

lucasew commented:
> `ayatana-indicator-sound` failure I assume is the same as in #369525? Any details on that yet?
> 
> Also note that this is a schema update, so running just the builds won't really reveal any issues. And `<pkg>.passthru.tests` which would involve VM tests that actually test if everything works are still not run automatically by `nixpkgs-review`.

Logs for ayatana-indicator-sound

<details>

```
Running phase: qtPreHook
@nix { "action": "setPhase", "phase": "qtPreHook" }
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/zs18wdrdgpilfg19093d1gzj1ks3k31s-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_LOCALEDIR=/nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1/share/doc/ayatana-indicator-sound -DCMAKE_INSTALL_INFODIR=/nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1/include -DCMAKE_INSTALL_SBINDIR=/nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/xcn9p4xxfbvlkpah7pwchpav4ab9d135-gcc-wrapper-14-20241116/bin/strip -DCMAKE_RANLIB=/nix/store/xcn9p4xxfbvlkpah7pwchpav4ab9d135-gcc-wrapper-14-20241116/bin/ranlib -DCMAKE_AR=/nix/store/xcn9p4xxfbvlkpah7pwchpav4ab9d135-gcc-wrapper-14-20241116/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1 -DENABLE_TESTS:BOOL=TRUE -DENABLE_LOMIRI_FEATURES:BOOL=TRUE -DGSETTINGS_LOCALINSTALL:BOOL=TRUE -DGSETTINGS_COMPILE:BOOL=TRUE
-- The C compiler identification is GNU 14.2.1
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/xcn9p4xxfbvlkpah7pwchpav4ab9d135-gcc-wrapper-14-20241116/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/xcn9p4xxfbvlkpah7pwchpav4ab9d135-gcc-wrapper-14-20241116/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /nix/store/0d4m43yp69lrm8imxbqgl9zxjwwz52jw-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Found Vala: /nix/store/v2kdxvkp36r7p9wagxbfm3qzhsqkcfyv-vala-0.56.17/bin/valac (found suitable version "0.56.17", minimum required is "0.20")
-- Checking for modules 'libpulse-mainloop-glib>=0.9.19;glib-2.0>=2.32;gio-unix-2.0>=2.25.13'
--   Found libpulse-mainloop-glib, version 17.0
--   Found glib-2.0, version 2.82.1
--   Found gio-unix-2.0, version 2.82.1
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Checking for modules 'gee-0.8;gio-2.0>=2.25.13;gio-unix-2.0;gthread-2.0;libxml-2.0;libnotify;accountsservice;libayatana-common;liblomiri-api>=0.1.1;lomiri-schemas'
--   Found gee-0.8, version 0.20.6
--   Found gio-2.0, version 2.82.1
--   Found gio-unix-2.0, version 2.82.1
--   Found gthread-2.0, version 2.82.1
--   Found libxml-2.0, version 2.13.5
--   Found libnotify, version 0.8.3
--   Found accountsservice, version 23.13.9
--   Found libayatana-common, version 0.9.10
--   Found liblomiri-api, version 0.2.0
--   Found lomiri-schemas, version 0.1.6
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Checking for module 'gobject-introspection-1.0'
--   Found gobject-introspection-1.0, version 1.82.0
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Found Gettext: /nix/store/9a46kza0vbb4vdsdmpzbdc7f31y7hs0i-gettext-0.22.5/bin/msgmerge (found version "0.22.5")
-- GSettings schemas will be installed locally.
-- GSettings schemas will be installed into /nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1/share/glib-2.0/schemas/
-- Installed GSettings schemas will be compiled.
-- Checking for module 'systemd'
--   Found systemd, version 256
-- /nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1/lib/systemd/user is the systemd user unit file install dir
-- Found Intltool: /nix/store/5kg3f63gkpwgdawnv9hvygcdis4arjvg-intltool-0.51.0/bin/intltool-update (found version "0.51.0")
-- Checking for module 'dbustest-1>=15.04.0'
--   Found dbustest-1, version 19.04.1
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found GTest: /nix/store/jjk5d24nfk0z87cg78cv5mzy2yj5z2fa-gtest-1.15.2-dev/lib/cmake/GTest/GTestConfig.cmake (found version "1.15.2")
-- Checking for module 'gtest'
--   Found gtest, version 1.15.2
[0m-- Using GTest v1.15.2 (parsed from pkg-config)[0m
-- Checking for module 'libgmenuharness'
--   Found libgmenuharness, version 0.1.4
-- Checking for module 'libqtdbustest-1'
--   Found libqtdbustest-1, version 1
-- Checking for module 'libqtdbusmock-1'
--   Found libqtdbusmock-1, version 1
-- Checking for modules 'glib-2.0>=2.26;gio-2.0>=2.26'
--   Found glib-2.0, version 2.82.1
--   Found gio-2.0, version 2.82.1
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Install prefix: /nix/store/1xc2fzn396r4hiz7z2rfszds8rwk1iyv-ayatana-indicator-sound-24.5.1
-- Unit tests: TRUE
-- Build with -Werror: OFF
-- Build with Lomiri features: TRUE
-- Configuring done (19.5s)
-- Generating done (1.3s)
[33mCMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025

[0m
-- Build files have been written to: /build/source/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: -j12 SHELL=/nix/store/5mh7kaj2fyv8mk4sfq1brwxgc02884wi-bash-5.2p37/bin/bash
[  0%] [34m[1mGenerating aa.gmo[0m
[  0%] [34m[1mGenerating accounts-service.vapi[0m
[  0%] [34m[1mGenerating am.gmo[0m
[  1%] [34m[1mGenerating af.gmo[0m
[  1%] [34m[1mGenerating an.gmo[0m
[  1%] [34m[1mGenerating ar.gmo[0m
[  1%] [34m[1mGenerating ast.gmo[0m
[  2%] [34m[1mGenerating as.gmo[0m
[  2%] Built target pofiles_1
[  2%] Built target pofiles_3
[  2%] [34m[1mGenerating bem.gmo[0m
[  2%] [34m[1mGenerating be.gmo[0m
[  2%] [34m[1mGenerating az.gmo[0m
[  2%] Built target pofiles_2
[  2%] Built target pofiles_6
[  2%] Built target pofiles_10
[  3%] [34m[1mGenerating bg.gmo[0m
[  3%] Built target pofiles_7
[  3%] Built target pofiles_5
[  3%] Built target pofiles_4
[  3%] Built target pofiles_9
[  3%] [34m[1mGenerating bo.gmo[0m
[  3%] Built target pofiles_8
[  3%] Built target pofiles_11
[  3%] [34m[1mGenerating bn.gmo[0m
[  4%] [34m[1mGenerating br.gmo[0m
[  4%] Built target pofiles_13
[  4%] [34m[1mGenerating ca.gmo[0m
[  5%] [34m[1mGenerating ca@valencia.gmo[0m
[  5%] [34m[1mGenerating ce.gmo[0m
[  5%] [34m[1mGenerating bs.gmo[0m
[  5%] [34m[1mGenerating ckb.gmo[0m
[  5%] [34m[1mGenerating cs.gmo[0m
[  5%] Built target pofiles_12
[  5%] Built target pofiles_14
[  5%] [34m[1mGenerating co.gmo[0m
[  5%] Built target pofiles_16
[  5%] [34m[1mGenerating crh.gmo[0m
[  5%] Built target pofiles_15
[  5%] Built target pofiles_18
[  5%] Built target pofiles_17
[  5%] Built target pofiles_19
[  5%] Built target pofiles_22
[  5%] Built target pofiles_20
[  5%] Built target pofiles_21
[  6%] [34m[1mGenerating csb.gmo[0m
[  6%] [34m[1mGenerating cv.gmo[0m
[  6%] Built target pofiles_23
[  6%] Built target pofiles_24
[  6%] [34m[1mGenerating cy.gmo[0m
[  6%] Built target pofiles_25
[  6%] [34m[1mGenerating dv.gmo[0m
[  6%] [34m[1mGenerating de.gmo[0m
[  7%] [34m[1mGenerating da.gmo[0m
[  8%] [34m[1mGenerating el.gmo[0m
[  8%] [34m[1mGenerating en_AU.gmo[0m
[  8%] Built target pofiles_28
[  8%] Built target pofiles_27
[  8%] [34m[1mGenerating en_CA.gmo[0m
[  8%] Built target pofiles_26
[  8%] Built target pofiles_29
[  9%] [34m[1mGenerating en_GB.gmo[0m
[  9%] Built target pofiles_30
[  9%] Built target pofiles_31
[ 10%] [34m[1mGenerating et.gmo[0m
[ 10%] [34m[1mGenerating eu.gmo[0m
[ 10%] [34m[1mGenerating eo.gmo[0m
[ 10%] [34m[1mGenerating es.gmo[0m
[ 10%] Built target pofiles_32
[ 10%] Built target pofiles_35
[ 10%] Built target pofiles_36
[ 10%] Built target pofiles_33
[ 10%] [34m[1mGenerating fa.gmo[0m
[ 10%] Built target pofiles_34
[ 10%] [34m[1mGenerating fr_CA.gmo[0m
[ 10%] [34m[1mGenerating fi.gmo[0m
[ 11%] [34m[1mGenerating fil.gmo[0m
[ 11%] [34m[1mGenerating fo.gmo[0m
[ 12%] [34m[1mGenerating fr.gmo[0m
[ 12%] Built target pofiles_37
[ 13%] [34m[1mGenerating fur.gmo[0m
[ 13%] Built target pofiles_38
[ 13%] [34m[1mGenerating frp.gmo[0m
[ 13%] Built target pofiles_40
[ 13%] [34m[1mGenerating fy.gmo[0m
[ 13%] Built target pofiles_42
[ 13%] Built target pofiles_39
[ 13%] Built target pofiles_41
[ 13%] [34m[1mGenerating ga.gmo[0m
[ 13%] [34m[1mGenerating gd.gmo[0m
[ 13%] Built target pofiles_43
[ 13%] Built target pofiles_44
[ 13%] Built target pofiles_45
[ 13%] Built target pofiles_46
[ 13%] [34m[1mGenerating gu.gmo[0m
[ 14%] [34m[1mGenerating gl.gmo[0m
Generation succeeded - 0 warning(s)
[ 14%] Built target pofiles_47
[ 15%] [34m[1mGenerating gv.gmo[0m
[ 15%] [34m[1mGenerating he.gmo[0m
[ 15%] Built target pofiles_49
[ 15%] [34m[1mGenerating hi.gmo[0m
[ 15%] Built target pofiles_48
[ 16%] [34m[1mGenerating hr.gmo[0m
[ 16%] [34m[1mGenerating ht.gmo[0m
[ 16%] [34m[1mGenerating hu.gmo[0m
[ 16%] [34m[1mGenerating hy.gmo[0m
[ 17%] [34m[1mGenerating ia.gmo[0m
[ 17%] Built target pofiles_51
[ 17%] Built target pofiles_50
[ 17%] Built target pofiles_52
[ 17%] Built target pofiles_54
[ 17%] [34m[1mGenerating id.gmo[0m
[ 17%] Built target pofiles_53
[ 17%] Built target pofiles_55
[ 17%] Built target pofiles_57
[ 17%] Built target pofiles_56
[ 17%] Built target pofiles_58
[ 17%] [34m[1mGenerating it.gmo[0m
[ 17%] [34m[1mGenerating is.gmo[0m
[ 17%] [34m[1mGenerating kk.gmo[0m
[ 17%] [34m[1mGenerating ka.gmo[0m
[ 17%] Built target pofiles_60
[ 17%] [34m[1mGenerating it_CARES.gmo[0m
[ 17%] Built target pofiles_59
[ 18%] [34m[1mGenerating km.gmo[0m
[ 18%] [34m[1mGenerating kl.gmo[0m
[ 18%] Built target pofiles_64
[ 18%] [34m[1mGenerating kmr.gmo[0m
[ 19%] [34m[1mGenerating ja.gmo[0m
[ 19%] [34m[1mGenerating kn.gmo[0m
[ 19%] Built target pofiles_61
[ 19%] Built target pofiles_63
[ 20%] [34m[1mGenerating ko.gmo[0m
[ 20%] Built target pofiles_65
[ 20%] Built target pofiles_66
[ 20%] Built target pofiles_67
[ 20%] Built target pofiles_62
[ 20%] Built target pofiles_68
[ 20%] Built target accounts-service
[ 21%] [34m[1mGenerating kw.gmo[0m
[ 21%] Built target pofiles_69
[ 21%] [34m[1mGenerating ku.gmo[0m
[ 21%] [34m[1mGenerating ky.gmo[0m
[ 21%] [34m[1mGenerating la.gmo[0m
[ 22%] [34m[1mGenerating lo.gmo[0m
[ 22%] [34m[1mGenerating lb.gmo[0m
[ 22%] Built target pofiles_71
[ 22%] Built target pofiles_70
[ 22%] Built target pofiles_72
[ 22%] Built target pofiles_73
[ 22%] Built target pofiles_75
[ 22%] Built target pofiles_74
[ 22%] [34m[1mGenerating lv.gmo[0m
[ 23%] [34m[1mGenerating mg.gmo[0m
[ 23%] [34m[1mGenerating lt.gmo[0m
[ 23%] [34m[1mGenerating mhr.gmo[0m
[ 24%] [34m[1mGenerating mk.gmo[0m
[ 24%] [34m[1mGenerating mi.gmo[0m
[ 24%] [34m[1mGenerating ml.gmo[0m
[ 24%] Built target pofiles_76
[ 24%] Built target pofiles_78
[ 24%] [34m[1mGenerating my.gmo[0m
[ 24%] Built target pofiles_77
[ 24%] Built target pofiles_79
[ 24%] Built target pofiles_81
[ 25%] [34m[1mGenerating ms.gmo[0m
[ 25%] [34m[1mGenerating nb.gmo[0m
[ 25%] Built target pofiles_82
[ 25%] [34m[1mGenerating mr.gmo[0m
[ 25%] Built target pofiles_80
[ 26%] [34m[1mGenerating ne.gmo[0m
[ 26%] Built target pofiles_85
[ 26%] [34m[1mGenerating nn.gmo[0m
[ 26%] Built target pofiles_84
[ 26%] Built target pofiles_86
[ 26%] [34m[1mGenerating nl.gmo[0m
[ 26%] Built target pofiles_87
[ 26%] Built target pofiles_83
[ 26%] [34m[1mGenerating os.gmo[0m
[ 27%] [34m[1mGenerating ny.gmo[0m
[ 27%] [34m[1mGenerating oc.gmo[0m
[ 27%] Built target pofiles_89
[ 28%] [34m[1mGenerating pa.gmo[0m
[ 28%] Built target pofiles_92
[ 28%] Built target pofiles_88
[ 28%] Built target pofiles_90
[ 29%] [34m[1mGenerating pt.gmo[0m
[ 29%] [34m[1mGenerating ps.gmo[0m
[ 29%] Built target pofiles_93
[ 29%] [34m[1mGenerating pl.gmo[0m
[ 29%] Built target pofiles_91
[ 29%] [34m[1mGenerating pt_BR.gmo[0m
[ 29%] Built target pofiles_96
[ 29%] [34m[1mGenerating ro.gmo[0m
[ 29%] [34m[1mGenerating sc.gmo[0m
[ 30%] [34m[1mGenerating sa.gmo[0m
[ 30%] [34m[1mGenerating ru.gmo[0m
[ 30%] Built target pofiles_95
[ 30%] [34m[1mGenerating sd.gmo[0m
[ 30%] Built target pofiles_94
[ 31%] [34m[1mGenerating sdh.gmo[0m
[ 31%] Built target pofiles_101
[ 31%] Built target pofiles_98
[ 31%] Built target pofiles_97
[ 31%] Built target pofiles_99
[ 31%] Built target pofiles_100
[ 31%] [34m[1mGenerating se.gmo[0m
[ 31%] Built target pofiles_102
[ 31%] Built target pofiles_103
[ 31%] [34m[1mGenerating shn.gmo[0m
[ 31%] [34m[1mGenerating sk.gmo[0m
[ 31%] Built target pofiles_104
[ 32%] [34m[1mGenerating si.gmo[0m
[ 32%] [34m[1mGenerating sq.gmo[0m
[ 32%] Built target pofiles_105
[ 32%] [34m[1mGenerating sr.gmo[0m
[ 32%] [34m[1mGenerating szl.gmo[0m
[ 32%] [34m[1mGenerating sl.gmo[0m
[ 32%] [34m[1mGenerating sv.gmo[0m
[ 33%] [34m[1mGenerating sw.gmo[0m
[ 33%] Built target pofiles_106
[ 33%] Built target pofiles_107
[ 33%] Built target pofiles_109
[ 33%] Built target pofiles_110
[ 33%] Built target pofiles_113
[ 33%] Built target pofiles_112
[ 33%] Built target pofiles_111
[ 33%] Built target pofiles_108
[ 33%] [34m[1mGenerating ta.gmo[0m
[ 33%] [34m[1mGenerating te.gmo[0m
[ 34%] [34m[1mGenerating ta_LK.gmo[0m
[ 34%] Built target pofiles_114
[ 34%] [34m[1mGenerating tg.gmo[0m
[ 34%] [34m[1mGenerating ur.gmo[0m
[ 34%] Built target pofiles_116
[ 34%] Built target pofiles_115
[ 34%] [34m[1mGenerating th.gmo[0m
[ 34%] [34m[1mGenerating tr.gmo[0m
[ 35%] [34m[1mGenerating ti.gmo[0m
[ 35%] Built target pofiles_118
[ 35%] Built target pofiles_124
[ 36%] [34m[1mGenerating tt.gmo[0m
[ 36%] [34m[1mGenerating uk.gmo[0m
[ 36%] Built target pofiles_119
[ 36%] [34m[1mGenerating ug.gmo[0m
[ 36%] [34m[1mGenerating wo.gmo[0m
[ 36%] Built target pofiles_117
[ 36%] [34m[1mGenerating vi.gmo[0m
[ 36%] [34m[1mGenerating zh_HK.gmo[0m
[ 37%] [34m[1mGenerating wae.gmo[0m
[ 37%] Built target pofiles_131
[ 37%] Built target pofiles_129
[ 37%] Built target pofiles_121
[ 37%] Built target pofiles_127
[ 37%] [34m[1mGenerating zh_LATN@pinyin.gmo[0m
[ 37%] [34m[1mGenerating vec.gmo[0m
[ 38%] [34m[1mGenerating uz.gmo[0m
[ 38%] Built target pofiles_123
[ 38%] [34m[1mGenerating home.created[0m
[ 38%] Built target pofiles_120
[ 38%] Built target pofiles_122
[ 38%] [34m[1mGenerating gschemas.compiled[0m
[ 38%] Built target pofiles_132
File does not exist
[ 39%] [34m[1mGenerating zh_CN.gmo[0m
[ 39%] Built target pofiles_126
[ 39%] Built target pofiles_128
[ 39%] Built target pofiles_125
[ 39%] [34m[1mGenerating zh_TW.gmo[0m
[ 39%] Built target test-home
[ 39%] Built target gschemas-compiled
[ 39%] Built target pofiles_130
[ 40%] [32mBuilding CXX object tests/CMakeFiles/pulse-mock.dir/pa-mock.cpp.o[0m
[ 40%] Built target pofiles_133
[ 41%] [34m[1mGenerating a fast vapi for warn-notification.vala[0m
[ 41%] [32mBuilding C object tests/CMakeFiles/name-watch-test.dir/__/src/bus-watch-namespace.c.o[0m
[ 42%] [32mBuilding CXX object tests/CMakeFiles/name-watch-test.dir/name-watch-test.cc.o[0m
[ 42%] [34m[1mPrecompiling accounts-service-access.vala[0m
[ 42%] Built target sound-indicator-dbus-interfaces_autogen_timestamp_deps
[ 43%] [34m[1mGenerating a fast vapi for accounts-service-access.vala[0m
[ 44%] [34m[1mPrecompiling accounts-service-privacy-settings.vala[0m
[ 44%] Built target accounts-service-sound_autogen_timestamp_deps
[ 44%] Built target media-player-mpris-mock_autogen_timestamp_deps
[ 44%] [32mBuilding CXX object tests/CMakeFiles/indicator-test.dir/indicator-test.cc.o[0m
[ 44%] Built target media-player-mpris-mock-update_autogen_timestamp_deps
[ 44%] [34m[1mGenerating a fast vapi for accounts-service-privacy-settings.vala[0m
[ 44%] [34m[1mPrecompiling accounts-service-sound-settings.vala[0m
[ 44%] [34m[1mAutomatic MOC for target sound-indicator-dbus-interfaces[0m
[ 44%] [34m[1mGenerating a fast vapi for accounts-service-sound-settings.vala[0m
[ 44%] [34m[1mAutomatic MOC for target accounts-service-sound[0m
[ 44%] [34m[1mPrecompiling accounts-service-system-sound-settings.vala[0m
[ 45%] [34m[1mAutomatic MOC for target media-player-mpris-mock[0m
[ 45%] [34m[1mGenerating a fast vapi for accounts-service-system-sound-settings.vala[0m
[ 45%] [34m[1mGenerating a fast vapi for greeter-broadcast.vala[0m
[ 45%] Built target sound-indicator-dbus-interfaces_autogen
[ 45%] [34m[1mGenerating a fast vapi for media-player.vala[0m
[ 45%] [34m[1mAutomatic MOC for target media-player-mpris-mock-update[0m
[ 45%] [34m[1mGenerating a fast vapi for mpris2-interfaces.vala[0m
[ 45%] Built target media-player-mpris-mock-update_autogen
[ 45%] [34m[1mGenerating a fast vapi for accounts-service-user.vala[0m
[ 45%] [34m[1mGenerating stream_restore_interface.cpp, stream_restore_interface.h[0m
[ 45%] [34m[1mPrecompiling freedesktop-interfaces.vala[0m
[ 46%] [34m[1mGenerating dbus_accounts_interface.cpp, dbus_accounts_interface.h[0m
[ 46%] [34m[1mGenerating dbus_accountssound_interface.cpp, dbus_accountssound_interface.h[0m
[ 47%] [34m[1mGenerating dbus_menus_interface.cpp, dbus_menus_interface.h[0m
[ 47%] [34m[1mGenerating dbus_notifications_interface.cpp, dbus_notifications_interface.h[0m
[ 47%] [34m[1mGenerating dbus_properties_interface.cpp, dbus_properties_interface.h[0m
[ 48%] [34m[1mGenerating a fast vapi for freedesktop-interfaces.vala[0m
[ 48%] [34m[1mGenerating stream_restore_interface.moc[0m
[ 48%] [34m[1mPrecompiling greeter-broadcast.vala[0m
[ 49%] [34m[1mGenerating a fast vapi for notification.vala[0m
[ 49%] Built target accounts-service-sound_autogen
[ 50%] [34m[1mGenerating testplayers[0m
[ 50%] [34m[1mGenerating dbus_accounts_interface.moc[0m
[ 50%] Built target media-player-mpris-mock_autogen
[ 50%] [34m[1mGenerating a fast vapi for info-notification.vala[0m
[ 51%] [34m[1mGenerating a fast vapi for service.vala[0m
[ 51%] [34m[1mGenerating a fast vapi for options.vala[0m
[ 51%] [34m[1mGenerating MediaPlayerMprisMockInterface.cpp, MediaPlayerMprisMockInterface.h[0m
[ 51%] [34m[1mGenerating accountsservice_sound_interface.cpp, accountsservice_sound_interface.h[0m
[ 51%] [34m[1mGenerating a fast vapi for options-gsettings.vala[0m
[ 52%] [34m[1mGenerating AccountsMockAdaptor.cpp, AccountsMockAdaptor.h[0m
[ 52%] [34m[1mGenerating a fast vapi for volume-control.vala[0m
[ 52%] [34m[1mGenerating dbus_accountssound_interface.moc[0m
[ 52%] [34m[1mGenerating MediaPlayerMprisMockInterface.moc[0m
[ 52%] [34m[1mGenerating AccountsServiceSoundMockAdaptor.cpp, AccountsServiceSoundMockAdaptor.h[0m
[ 52%] [34m[1mGenerating a fast vapi for volume-warning.vala[0m
[ 52%] [34m[1mGenerating a fast vapi for volume-control-pulse.vala[0m
[ 52%] [34m[1mGenerating dbus_menus_interface.moc[0m
[ 52%] [34m[1mGenerating a fast vapi for volume-warning-pulse.vala[0m
[ 53%] [34m[1mGenerating a fast vapi for media-player-mpris.vala[0m
[ 53%] [34m[1mGenerating accountsservice_sound_interface.moc[0m
[ 53%] [34m[1mGenerating dbus_notifications_interface.moc[0m
[ 53%] [34m[1mGenerating a fast vapi for media-player-user.vala[0m
[ 54%] [34m[1mGenerating dbus_properties_interface.moc[0m
[ 55%] [34m[1mGenerating a fast vapi for media-player-list.vala[0m
[ 55%] [34m[1mGenerating a fast vapi for media-player-list-mpris.vala[0m
[ 55%] [34m[1mGenerating a fast vapi for media-player-list-greeter.vala[0m
[ 55%] [34m[1mGenerating testplayers[0m
[ 56%] [34m[1mGenerating a fast vapi for sound-menu.vala[0m
[ 56%] [34m[1mPrecompiling info-notification.vala[0m
[ 56%] [34m[1mGenerating MediaPlayer2MockAdaptor.cpp, MediaPlayer2MockAdaptor.h[0m
[ 57%] [34m[1mGenerating MediaPlayerMprisMockAdaptor.cpp, MediaPlayerMprisMockAdaptor.h[0m
[ 57%] [34m[1mGenerating AccountsMockAdaptor.moc[0m
[ 57%] [34m[1mGenerating MediaPlayer2MockAdaptor.moc[0m
[ 58%] [34m[1mPrecompiling media-player-list-greeter.vala[0m
[ 58%] [34m[1mGenerating MediaPlayerMprisMockAdaptor.moc[0m
[ 58%] [32mBuilding CXX object tests/service-mocks/media-player-mpris-mock/CMakeFiles/media-player-mpris-mock-update.dir/media-player-mpris-mock-update_autogen/mocs_compilation.cpp.o[0m
[ 59%] [34m[1mGenerating AccountsServiceSoundMockAdaptor.moc[0m
[ 60%] [32mBuilding CXX object tests/service-mocks/media-player-mpris-mock/CMakeFiles/media-player-mpris-mock-update.dir/MediaPlayerMprisMockInterface.cpp.o[0m
[ 60%] [32mBuilding CXX object tests/dbus-types/CMakeFiles/sound-indicator-dbus-interfaces.dir/sound-indicator-dbus-interfaces_autogen/mocs_compilation.cpp.o[0m
[ 60%] [34m[1mPrecompiling media-player-list-mpris.vala[0m
[ 60%] [32mBuilding CXX object tests/dbus-types/CMakeFiles/sound-indicator-dbus-interfaces.dir/stream_restore_interface.cpp.o[0m
[ 61%] [32mBuilding CXX object tests/dbus-types/CMakeFiles/sound-indicator-dbus-interfaces.dir/dbus_properties_interface.cpp.o[0m
[ 61%] [32mBuilding CXX object tests/service-mocks/media-player-mpris-mock/CMakeFiles/media-player-mpris-mock.dir/media-player-mpris-mock_autogen/mocs_compilation.cpp.o[0m
[ 61%] [34m[1mPrecompiling media-player-list.vala[0m
[ 61%] [32mBuilding CXX object tests/service-mocks/accounts-mock/CMakeFiles/accounts-service-sound.dir/accounts-service-sound_autogen/mocs_compilation.cpp.o[0m
[ 61%] [32mBuilding CXX object tests/service-mocks/media-player-mpris-mock/CMakeFiles/media-player-mpris-mock.dir/MediaPlayerMprisMockAdaptor.cpp.o[0m
[ 61%] [34m[1mPrecompiling media-player-user.vala[0m
[ 62%] [34m[1mPrecompiling media-player-mpris.vala[0m
[ 62%] [34m[1mPrecompiling media-player.vala[0m
[ 63%] [32mBuilding CXX object tests/service-mocks/media-player-mpris-mock/CMakeFiles/media-player-mpris-mock.dir/MediaPlayer2MockAdaptor.cpp.o[0m
[ 63%] [34m[1mPrecompiling mpris2-interfaces.vala[0m
[ 63%] [34m[1mPrecompiling notification.vala[0m
[ 64%] [34m[1mPrecompiling options-gsettings.vala[0m
[ 64%] [32m[1mLinking CXX shared library libpulse-mock.so[0m
[ 64%] [34m[1mPrecompiling options.vala[0m
[ 64%] Built target pulse-mock
[ 64%] [32mBuilding CXX object tests/service-mocks/media-player-mpris-mock/CMakeFiles/media-player-mpris-mock-update.dir/player-update.cpp.o[0m
[ 64%] [34m[1mPrecompiling service.vala[0m
[ 65%] [34m[1mPrecompiling sound-menu.vala[0m
[ 65%] [34m[1mPrecompiling volume-control-pulse.vala[0m
[ 65%] [34m[1mPrecompiling volume-control.vala[0m
[ 65%] [32m[1mLinking CXX executable name-watch-test[0m
[ 65%] [32mBuilding CXX object tests/service-mocks/media-player-mpris-mock/CMakeFiles/media-player-mpris-mock.dir/MediaPlayerMprisMock.cpp.o[0m
[ 65%] [32mBuilding CXX object tests/service-mocks/accounts-mock/CMakeFiles/accounts-service-sound.dir/accountsservice_sound_interface.cpp.o[0m
[ 65%] Built target name-watch-test
[ 65%] [32mBuilding CXX object tests/service-mocks/media-player-mpris-mock/CMakeFiles/media-player-mpris-mock.dir/__/DBusPropertiesNotifier.cpp.o[0m
[ 65%] [34m[1mPrecompiling volume-warning-pulse.vala[0m
[ 66%] [34m[1mPrecompiling volume-warning.vala[0m
[ 67%] [34m[1mPrecompiling warn-notification.vala[0m
[ 67%] [34m[1mPrecompiling accounts-service-user.vala[0m
[ 67%] [34m[1mGenerating vapi/headers/symbols[0m
[ 67%] [32mBuilding CXX object tests/dbus-types/CMakeFiles/sound-indicator-dbus-interfaces.dir/dbus_accounts_interface.cpp.o[0m
[ 68%] [32mBuilding CXX object tests/service-mocks/accounts-mock/CMakeFiles/accounts-service-sound.dir/AccountsServiceSoundMockAdaptor.cpp.o[0m
[ 68%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/notification.c.o[0m
[ 68%] [32mBuilding CXX object tests/service-mocks/media-player-mpris-mock/CMakeFiles/media-player-mpris-mock.dir/main.cpp.o[0m
[ 68%] [32mBuilding CXX object tests/dbus-types/CMakeFiles/sound-indicator-dbus-interfaces.dir/dbus_accountssound_interface.cpp.o[0m
[ 68%] [32mBuilding CXX object tests/dbus-types/CMakeFiles/sound-indicator-dbus-interfaces.dir/dbus_menus_interface.cpp.o[0m
[ 69%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/info-notification.c.o[0m
[ 70%] [32mBuilding CXX object tests/dbus-types/CMakeFiles/sound-indicator-dbus-interfaces.dir/dbus_notifications_interface.cpp.o[0m
[ 70%] [32mBuilding CXX object tests/service-mocks/accounts-mock/CMakeFiles/accounts-service-sound.dir/AccountsMockAdaptor.cpp.o[0m
[ 70%] [32mBuilding CXX object tests/dbus-types/CMakeFiles/sound-indicator-dbus-interfaces.dir/pulseaudio-volume.cpp.o[0m
[ 70%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/warn-notification.c.o[0m
[ 71%] [32m[1mLinking CXX executable indicator-test[0m
[ 71%] Built target indicator-test
[ 71%] [32mBuilding CXX object tests/service-mocks/accounts-mock/CMakeFiles/accounts-service-sound.dir/AccountsServiceSoundMock.cpp.o[0m
[ 71%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/service.c.o[0m
[ 71%] [32m[1mLinking CXX executable media-player-mpris-mock-update[0m
[ 71%] Built target media-player-mpris-mock-update
[ 72%] [32mBuilding CXX object tests/service-mocks/accounts-mock/CMakeFiles/accounts-service-sound.dir/AccountsMock.cpp.o[0m
[ 72%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/options.c.o[0m
[ 72%] [32mBuilding CXX object tests/dbus-types/CMakeFiles/sound-indicator-dbus-interfaces.dir/dbus-action-result.cpp.o[0m
[ 72%] [32mBuilding CXX object tests/service-mocks/accounts-mock/CMakeFiles/accounts-service-sound.dir/__/DBusPropertiesNotifier.cpp.o[0m
[ 73%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/options-gsettings.c.o[0m
[ 73%] [32mBuilding CXX object tests/service-mocks/accounts-mock/CMakeFiles/accounts-service-sound.dir/main.cpp.o[0m
[ 73%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/volume-control.c.o[0m
[ 73%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/accounts-service-access.c.o[0m
[ 74%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/volume-control-pulse.c.o[0m
[ 74%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/volume-warning.c.o[0m
[ 74%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/volume-warning-pulse.c.o[0m
[ 75%] [32m[1mLinking CXX executable media-player-mpris-mock[0m
[ 75%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/media-player.c.o[0m
[ 76%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/media-player-mpris.c.o[0m
[ 76%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/media-player-user.c.o[0m
[ 76%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/media-player-list.c.o[0m
[ 77%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/media-player-list-mpris.c.o[0m
[ 78%] Built target media-player-mpris-mock
[ 78%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/media-player-list-greeter.c.o[0m
[ 79%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/freedesktop-interfaces.c.o[0m
[ 79%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/mpris2-interfaces.c.o[0m
[ 79%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/sound-menu.c.o[0m
[ 79%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/accounts-service-user.c.o[0m
[ 79%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/accounts-service-sound-settings.c.o[0m
[ 80%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/accounts-service-privacy-settings.c.o[0m
[ 80%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/accounts-service-system-sound-settings.c.o[0m
[ 80%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/greeter-broadcast.c.o[0m
[ 81%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-lib.dir/bus-watch-namespace.c.o[0m
[ 81%] [32m[1mLinking C static library libindicator-sound-service-lib.a[0m
[ 81%] Built target indicator-sound-service-lib
[ 81%] [34m[1mGenerating a fast vapi for volume-warning-mock.vala[0m
[ 81%] [34m[1mPrecompiling media-player-list-mock.vala[0m
[ 82%] [34m[1mGenerating a fast vapi for options-mock.vala[0m
[ 82%] [34m[1mGenerating a fast vapi for media-player-list-mock.vala[0m
[ 82%] [34m[1mGenerating a fast vapi for media-player-mock.vala[0m
[ 82%] [34m[1mPrecompiling media-player-mock.vala[0m
[ 83%] [34m[1mPrecompiling options-mock.vala[0m
[ 83%] [34m[1mGenerating a fast vapi for volume-control-mock.vala[0m
[ 83%] [32mBuilding C object src/CMakeFiles/indicator-sound-service-bin.dir/main.c.o[0m
[ 83%] [34m[1mPrecompiling volume-control-mock.vala[0m
[ 83%] [34m[1mPrecompiling volume-warning-mock.vala[0m
[ 84%] [34m[1mGenerating vapi/headers/symbols[0m
[ 85%] [32m[1mLinking C executable ayatana-indicator-sound-service[0m
[ 85%] [32mBuilding C object tests/CMakeFiles/vala-mocks-lib.dir/media-player-mock.c.o[0m
[ 85%] [32mBuilding C object tests/CMakeFiles/vala-mocks-lib.dir/options-mock.c.o[0m
[ 86%] [32mBuilding C object tests/CMakeFiles/vala-mocks-lib.dir/media-player-list-mock.c.o[0m
[ 86%] [32mBuilding C object tests/CMakeFiles/vala-mocks-lib.dir/volume-control-mock.c.o[0m
[ 87%] [32mBuilding C object tests/CMakeFiles/vala-mocks-lib.dir/volume-warning-mock.c.o[0m
[ 88%] [32m[1mLinking CXX static library libsound-indicator-dbus-interfaces.a[0m
[ 89%] Built target sound-indicator-dbus-interfaces
[ 90%] [32m[1mLinking CXX executable accounts-service-sound[0m
[ 90%] Built target set-volume_autogen_timestamp_deps
[ 90%] Built target get-volume_autogen_timestamp_deps
[ 90%] Built target integration-tests_autogen_timestamp_deps
[ 90%] [34m[1mAutomatic MOC for target get-volume[0m
[ 90%] [34m[1mAutomatic MOC for target set-volume[0m
[ 90%] [34m[1mAutomatic MOC for target integration-tests[0m
[ 90%] Built target indicator-sound-service-bin
[ 90%] Built target set-volume_autogen
[ 90%] Built target get-volume_autogen
[ 91%] [32mBuilding CXX object tests/integration/CMakeFiles/set-volume.dir/set-volume_autogen/mocs_compilation.cpp.o[0m
[ 91%] [32mBuilding CXX object tests/integration/CMakeFiles/get-volume.dir/utils/dbus-pulse-volume.cpp.o[0m
[ 91%] [32mBuilding CXX object tests/integration/CMakeFiles/set-volume.dir/utils/dbus-pulse-volume.cpp.o[0m
[ 92%] [32mBuilding CXX object tests/integration/CMakeFiles/get-volume.dir/utils/get-volume.cpp.o[0m
[ 92%] [32mBuilding CXX object tests/integration/CMakeFiles/get-volume.dir/get-volume_autogen/mocs_compilation.cpp.o[0m
[ 92%] [32mBuilding CXX object tests/integration/CMakeFiles/set-volume.dir/utils/set-volume.cpp.o[0m
[ 92%] Built target accounts-service-sound
[ 92%] [32m[1mLinking C static library libvala-mocks-lib.a[0m
[ 92%] Built target vala-mocks-lib
[ 92%] [32mBuilding CXX object tests/CMakeFiles/accounts-service-user-test.dir/accounts-service-user.cc.o[0m
[ 93%] [32mBuilding CXX object tests/CMakeFiles/notifications-test.dir/notifications-test.cc.o[0m
[ 93%] [32mBuilding CXX object tests/CMakeFiles/volume-control-test.dir/volume-control-test.cc.o[0m
[ 93%] [32mBuilding CXX object tests/CMakeFiles/sound-menu-test.dir/sound-menu.cc.o[0m
[ 94%] [32mBuilding CXX object tests/CMakeFiles/greeter-list-test.dir/greeter-list.cc.o[0m
[ 94%] [32mBuilding CXX object tests/CMakeFiles/media-player-user-test.dir/media-player-user.cc.o[0m
[ 94%] Built target integration-tests_autogen
[ 94%] [32mBuilding CXX object tests/integration/CMakeFiles/integration-tests.dir/integration-tests_autogen/mocs_compilation.cpp.o[0m
[ 94%] [32mBuilding CXX object tests/integration/CMakeFiles/integration-tests.dir/indicator-sound-test-base.cpp.o[0m
[ 95%] [32mBuilding CXX object tests/integration/CMakeFiles/integration-tests.dir/test-indicator.cpp.o[0m
[ 96%] [32m[1mLinking CXX executable volume-control-test[0m
[ 96%] [32m[1mLinking CXX executable greeter-list-test[0m
[ 96%] Built target volume-control-test
[ 96%] [32mBuilding CXX object tests/integration/CMakeFiles/integration-tests.dir/utils/dbus-pulse-volume.cpp.o[0m
[ 96%] [32m[1mLinking CXX executable accounts-service-user-test[0m
[ 96%] Built target greeter-list-test
[ 96%] [32mBuilding CXX object tests/integration/CMakeFiles/integration-tests.dir/main.cpp.o[0m
[ 96%] Built target accounts-service-user-test
[ 97%] [32m[1mLinking CXX executable sound-menu-test[0m
[ 97%] Built target sound-menu-test
[ 97%] [32m[1mLinking CXX executable get-volume[0m
[ 98%] Built target get-volume
[ 99%] [32m[1mLinking CXX executable set-volume[0m
[ 99%] [32m[1mLinking CXX executable media-player-user-test[0m
[ 99%] Built target set-volume
[ 99%] Built target media-player-user-test
[ 99%] [32m[1mLinking CXX executable notifications-test[0m
[ 99%] Built target notifications-test
[100%] [32m[1mLinking CXX executable integration-tests[0m
[100%] Built target integration-tests
buildPhase completed in 3 minutes 24 seconds
Running phase: checkPhase
@nix { "action": "setPhase", "phase": "checkPhase" }
check flags: SHELL=/nix/store/5mh7kaj2fyv8mk4sfq1brwxgc02884wi-bash-5.2p37/bin/bash VERBOSE=y test
[36mRunning tests...[0m
/nix/store/38cffsqqx823crf1i4bcf6zz1qz1hgpd-cmake-3.30.5/bin/ctest --force-new-ctest-process 
Test project /build/source/build
      Start  1: name-watch-test
      Start  2: accounts-service-user-test-basic
      Start  3: accounts-service-user-test-player
      Start  4: volume-control-test
      Start  5: sound-menu-test
      Start  6: notifications-test
      Start  7: media-player-user-test-basic
      Start  8: media-player-user-test-dataset
      Start  9: media-player-user-test-timeout
      Start 10: greeter-list-test-basic
      Start 11: greeter-list-test-iterator
      Start 12: indicator-test
 1/13 Test  #5: sound-menu-test .....................   Passed    0.12 sec
 2/13 Test  #8: media-player-user-test-dataset ......   Passed    0.07 sec
 3/13 Test  #9: media-player-user-test-timeout ......   Passed    0.06 sec
      Start 13: integration-tests
 4/13 Test #11: greeter-list-test-iterator ..........   Passed    0.06 sec
 5/13 Test #13: integration-tests ...................   Passed    0.02 sec
 6/13 Test #12: indicator-test ......................   Passed    0.09 sec
 7/13 Test #10: greeter-list-test-basic .............   Passed    0.12 sec
 8/13 Test  #4: volume-control-test .................   Passed    0.22 sec
 9/13 Test  #7: media-player-user-test-basic ........   Passed    1.08 sec
10/13 Test  #2: accounts-service-user-test-basic ....   Passed    1.24 sec
11/13 Test  #6: notifications-test ..................   Passed    2.65 sec
12/13 Test  #3: accounts-service-user-test-player ...   Passed    2.77 sec
13/13 Test  #1: name-watch-test .....................***Failed   30.88 sec
Running main() from gmock_main.cc
[==========] Running 3 tests from 1 test suite.
[----------] Global test environment set-up.
[----------] 3 tests from NameWatchTest
[ RUN      ] NameWatchTest.BaseWatch
/build/source/tests/name-watch-test.cc:101: Failure
Expected equality of these values:
  callback_count.appeared
    Which is: 0
  2


(process:2478): GLib-GIO-CRITICAL **: 17:12:27.637: Error while sending AddMatch() message: The connection is closed

** (process:2478): WARNING **: 17:12:27.638: bus_watch_namespace: error calling org.freedesktop.DBus.ListNames: The connection is closed

(process:2478): GLib-GIO-WARNING **: 17:12:57.791: Weak notify timeout, object ref_count=3
[  FAILED  ] NameWatchTest.BaseWatch (30284 ms)
[ RUN      ] NameWatchTest.NonMatches
[       OK ] NameWatchTest.NonMatches (243 ms)
[ RUN      ] NameWatchTest.StartupNames
[       OK ] NameWatchTest.StartupNames (332 ms)
[----------] 3 tests from NameWatchTest (30860 ms total)

[----------] Global test environment tear-down
[==========] 3 tests from 1 test suite ran. (30860 ms total)
[  PASSED  ] 2 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] NameWatchTest.BaseWatch

 1 FAILED TEST


92% tests passed[0;0m, [0;31m1 tests failed[0;0m out of 13

Total Test time (real) =  30.89 sec

The following tests FAILED:
	[0;31m  1 - name-watch-test (Failed)[0;0m
Errors while running CTest
make: *** [Makefile:71: test] Error 8
```

</details>

OPNA2608 commented:
> ```
> (process:2478): GLib-GIO-CRITICAL **: 17:12:27.637: Error while sending AddMatch() message: The connection is closed
> 
> ** (process:2478): WARNING **: 17:12:27.638: bus_watch_namespace: error calling org.freedesktop.DBus.ListNames: The connection is closed
> ```

… seems to be more evidence towards https://github.com/NixOS/nixpkgs/pull/369525#issuecomment-2567029178

> […] assuming this is maybe an issue with DBus getting spun up & shut down by multiple tests with unfortunate timing […]

Still dunno how reproducible this failure is for you, but I'll add a commit which should *hopefully* work around this. Please retry building the package afew times once it's pushed, to make sure the issue is gone for you.

OPNA2608 committed:
ayatana-indicator-sound: Disable parallel testing

Spins up & talks with D-Bus in multiple tests, parallelism gets tests talking to each others' D-Bus daemons
and failing when the bus they're talking to gets shut down by another test.

lucasew commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 370966`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 38 packages built:</summary>
  <ul>
    <li>ayatana-indicator-bluetooth</li>
    <li>ayatana-indicator-datetime</li>
    <li>ayatana-indicator-display</li>
    <li>ayatana-indicator-power</li>
    <li>ayatana-indicator-session</li>
    <li>ayatana-indicator-sound</li>
    <li>libayatana-common</li>
    <li>lomiri.lomiri</li>
    <li>lomiri.lomiri-app-launch</li>
    <li>lomiri.lomiri-app-launch.dev</li>
    <li>lomiri.lomiri-app-launch.doc</li>
    <li>lomiri.lomiri-camera-app</li>
    <li>lomiri.lomiri-clock-app</li>
    <li>lomiri.lomiri-content-hub</li>
    <li>lomiri.lomiri-content-hub.dev</li>
    <li>lomiri.lomiri-content-hub.doc</li>
    <li>lomiri.lomiri-content-hub.examples</li>
    <li>lomiri.lomiri-docviewer-app</li>
    <li>lomiri.lomiri-filemanager-app</li>
    <li>lomiri.lomiri-gallery-app</li>
    <li>lomiri.lomiri-indicator-network</li>
    <li>lomiri.lomiri-indicator-network.dev</li>
    <li>lomiri.lomiri-indicator-network.doc</li>
    <li>lomiri.lomiri-mediaplayer-app</li>
    <li>lomiri.lomiri-push-qml</li>
    <li>lomiri.lomiri-schemas</li>
    <li>lomiri.lomiri-session</li>
    <li>lomiri.lomiri-system-settings</li>
    <li>lomiri.lomiri-system-settings-unwrapped</li>
    <li>lomiri.lomiri-system-settings-unwrapped.dev</li>
    <li>lomiri.lomiri-url-dispatcher</li>
    <li>lomiri.lomiri-url-dispatcher.dev</li>
    <li>lomiri.lomiri-url-dispatcher.lib</li>
    <li>lomiri.morph-browser</li>
    <li>lomiri.qtmir</li>
    <li>lomiri.qtmir.dev</li>
    <li>lomiri.telephony-service</li>
    <li>lomiri.teleports</li>
  </ul>
</details>

OPNA2608 commented:
So… I assume the error doesn't reproduce at all for you anymore? Or if it was flaky, then you did afew rebuilds as asked to make *sure* this fixes it?

lucasew commented:
> So… I assume the error doesn't reproduce at all for you anymore? Or if it was flaky, then you did afew rebuilds as asked to make _sure_ this fixes it?

I did requeue the nixpkgs-review job. That should be just a data race that disabling parallel tests solved.

OPNA2608 committed:
lomiri.morph-browser: Modernise, help with test timeout, add doc output

OPNA2608 committed:
lomiri.lomiri-ui-toolkit: Reinstate problematic test marking

Turns out that not every test is running well yet:
https://gitlab.com/ubports/development/core/lomiri-ui-toolkit/-/issues/43

OPNA2608 commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 370966`

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 46 packages built:</summary>
  <ul>
    <li>ayatana-indicator-bluetooth</li>
    <li>ayatana-indicator-datetime</li>
    <li>ayatana-indicator-display</li>
    <li>ayatana-indicator-power</li>
    <li>ayatana-indicator-session</li>
    <li>ayatana-indicator-sound</li>
    <li>deltatouch</li>
    <li>libayatana-common</li>
    <li>lomiri.lomiri</li>
    <li>lomiri.lomiri-app-launch</li>
    <li>lomiri.lomiri-app-launch.dev</li>
    <li>lomiri.lomiri-app-launch.doc</li>
    <li>lomiri.lomiri-calculator-app</li>
    <li>lomiri.lomiri-camera-app</li>
    <li>lomiri.lomiri-clock-app</li>
    <li>lomiri.lomiri-content-hub</li>
    <li>lomiri.lomiri-content-hub.dev</li>
    <li>lomiri.lomiri-content-hub.doc</li>
    <li>lomiri.lomiri-content-hub.examples</li>
    <li>lomiri.lomiri-docviewer-app</li>
    <li>lomiri.lomiri-filemanager-app</li>
    <li>lomiri.lomiri-gallery-app</li>
    <li>lomiri.lomiri-indicator-network</li>
    <li>lomiri.lomiri-indicator-network.dev</li>
    <li>lomiri.lomiri-indicator-network.doc</li>
    <li>lomiri.lomiri-mediaplayer-app</li>
    <li>lomiri.lomiri-push-qml</li>
    <li>lomiri.lomiri-schemas</li>
    <li>lomiri.lomiri-session</li>
    <li>lomiri.lomiri-system-settings</li>
    <li>lomiri.lomiri-system-settings-unwrapped</li>
    <li>lomiri.lomiri-system-settings-unwrapped.dev</li>
    <li>lomiri.lomiri-terminal-app</li>
    <li>lomiri.lomiri-ui-extras</li>
    <li>lomiri.lomiri-ui-toolkit</li>
    <li>lomiri.lomiri-ui-toolkit.dev</li>
    <li>lomiri.lomiri-ui-toolkit.doc</li>
    <li>lomiri.lomiri-url-dispatcher</li>
    <li>lomiri.lomiri-url-dispatcher.dev</li>
    <li>lomiri.lomiri-url-dispatcher.lib</li>
    <li>lomiri.morph-browser</li>
    <li>lomiri.morph-browser.doc</li>
    <li>lomiri.qtmir</li>
    <li>lomiri.qtmir.dev</li>
    <li>lomiri.telephony-service</li>
    <li>lomiri.teleports</li>
  </ul>
</details>

---

Assuming there are no more complaints about this. Merging, so I can go ahead with bumping some of the modified packages in separate PRs.
